src/routes/response_formats.rs

```diff
@@ -1,9 +1,20 @@
-use axum::{response::Html, routing::get, Router};
+use axum::{response::Html, response::IntoResponse, routing::get, Router,
+  http::{StatusCode, header::{self}}};
 
 const UTF8_PAGE: &str = include_str!("../templates/utf8.html");
+const XML_PAGE: &str = include_str!("../templates/sample.xml");
 
 pub fn routes() -> Router {
     Router::new().route("/encoding/utf8", get(utf8))
+    .route("/xml", get(xml))
+}
+
+async fn xml() -> impl IntoResponse {
+    (
+        StatusCode::OK,
+        [(header::CONTENT_TYPE, "text/xml")],
```

Comment:
The original httpbin project uses `"application/xml"` here, which I believe isn't in the `Mime` codebase we're using (same as `"application/yaml"`).  Should we maintain consistency with the original httpbin?

> **Note**
> Instead of `"text/xml"`, you can use `mime::TEXT_XML.essence_str()`?  See: https://docs.rs/mime/latest/mime/struct.Mime.html#method.essence_str
> 
> In fact, all of the `Mime` `as_ref()` instances should probably be `essence_str()`, even though they are technically working out to be identical.  You can go for consistence with `as_ref()` if you like and we can do another PR to change them to `essence_str()` later.  Up to you.

Reply:
I'm happy to break compatibility with the Python codebase on minor things.